Fix caldb issue with bias lines >= 12

jlashner opened the pull request:
This resolves #1012 , where update_smurf_caldbs fails when the bias voltage on lines >= 12 are not nan in the bias step analysis.

mhasself reviewed (changes requested):
It's helpful to Future Us in cases like this to add a note explaining that the redundant check really is necessary. E.g.

`# Usually bias lines >= 12 show up as nan but we have seen cases where they are populated and need to be rejected based on count`

jlashner commented:
Thanks Matthew, I added a comment, and also noticed we would need to make the same change in the new `update_det_cal` script, so I added this check there as well.

mhasself reviewed (approved):
Thanks for quick fix!  Please remember to squash on merge.
